travis_build.sh

```diff
@@ -30,10 +30,14 @@ git_clean() {
 
 build() {
     ./build.sh
-    CONFIGURE=$(readlink -f configure)
+    CONFIGURE=$(realpath configure)
```

Comment:
"readlink" in mac has no "-f" option. Here I replace with "readpath" because configure is not symbolic link in our case.

Reply:
didn't realise that's the only use of `readlink`!
You can also do `$TRAVIS_BUILD_DIR/configure`. See [docs](https://docs.travis-ci.com/user/environment-variables/).